README.md

```diff
@@ -66,11 +66,13 @@ Before start using *slacktee*, please set following variables in the script conf
 *slacktee* reads the global configuration (/etc/slacktee.conf) first, then reads your local configuration (~/.slacktee).
 You can set up your local configuration file using interactive setup mode (--setup option).
 
-For more details about tokens, visit [Slack's API page](https://api.slack.com/).
+In order to retrieve the bot token, you need to add a bot into your workspace through [Slack App Directory](https://cks-world.slack.com/apps/A0F7YS25R-bots).
+Once you add a bot, you can find 'API Token' in the configuration page of the bot.
+
```

Comment:
Maybe I would word it in steps. coz its either or. so the comment can say:
You would need an authentication token for the bot. It could be generated in 2 ways
1. (Preferred way) Follow steps listed in [creating a Slack App](https://api.slack.com/slack-apps#creating_apps).
Next, create a bot user for your app and give the following 3 permissions to your app - `chat:write:bot`, `files:write:user` and `bot`. More information about the permission scopes can be found at [permission scopes](https://api.slack.com/docs/oauth-scopes)
2. Add a bot into your workspace through [Slack App Directory](https://cks-world.slack.com/apps/A0F7YS25R-bots). You can now find 'API Token' in the configuration page of the bot.

Reply:
Wow, great suggestion! I will update README!

Reply:
Updated!